guizero/LabelBox.py

```diff
@@ -0,0 +1,114 @@
+from tkinter import LabelFrame
+from . import utilities as utils
+from .base import ContainerTextWidget
+from .event import EventManager
+
+class LabelBox(ContainerTextWidget):
+
+    def __init__(
+        self,
+        master,
+        text="",
+        layout="auto",
+        grid=None,
+        align=None,
+        visible=True,
+        enabled=None,
+        width=None,
+        height=None,
+        border=None):
+        """
+        Creates a LabelBox
+
+        :param Container master:
+            The Container (App, LabelBox, etc) the LabelBox will belong too.
+
+        :param string text:
+            The text to be displayed on the box label.
+
+        :param string layout:
+            The layout the LabelBox should use "auto" or "grid. Defaults to "auto".
+
+        :param List grid:
+            Grid co-ordinates for the widget, required if the master layout
+            is 'grid', defaults to `None`.
+
+        :param string align:
+            How to align the widget within the grid, defaults to `None`.
+
+        :param callback args:
+            A list of arguments to pass to the widgets `command`, defaults to
+            `None`.
+
+        :param bool visible:
+            If the widget should be visible, defaults to `True`.
+
+        :param bool enabled:
+            If the widget should be enabled, defaults to `None`. If `None`
+            the value is inherited from the master.
+
+        :param int width:
+            The starting width of the widget. Defaults to `None` and will auto
+            size. If not `None`, both the width and height need to be set.
+
+        :param int height:
+            The starting height of the widget. Defaults to `None` and will auto
+            size. If not `None`, both the width and height need to be set.
+
+        :param int border:
+            Sets the border thickness. `0` or `False` is no border. `True` or
+            value > 1 sets a border. The default is `None`.
```

Comment:
Similar question - what is the default behaviour of LabelFrame? Does it have a border? If so, should this be 1?!